pkg/pillar/types/global_test.go

```diff
@@ -126,6 +129,84 @@ func TestAddStringItem(t *testing.T) {
 		assert.Equal(t, test.expectedVal, specMap.GlobalSettings[test.key].StringDefault)
 	}
 }
+
+func TestNewConfigItemSpecMap(t *testing.T) {
+	// log.SetLevel(log.DebugLevel)
+	specMap := NewConfigItemSpecMap()
```

Comment:
So this is testing using the map we have in the product,right?

Reply:
Yes..